submissions/ArthurGorbenko/tiny-js-world/index.js

```diff
@@ -0,0 +1,94 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/ArthurGorbenko/a-tiny-JS-world/blob/gh-pages/index.js
+   Web app: https://arthurgorbenko.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+const dog = {
+    type : "dog",
+    name : "Barley",
+    gender : "male",
+    legs : 4,
+    hasTail : true,
+    saying : "gav-gav"
+}
+
+const cat = {
+    type : "cat",
+    name : "Rude",
+    gender : "male",
+    legs : 4,
+    hasTail : true,
+    saying : "Kottans are the best!"
+}
+
+const man = {
+    type : "human",
+    name : "Alex",
+    gender : "male",
+    legs : 2,
+    hasTail : false,
+    saying : "I want to start learning frontend development!"
+}
+
+const woman = {
+    type : "human",
+    name : "Meggy",
+    gender : "female",
+    legs : 2,
+    hasTail : false,
+    saying : "Hello,Alex."
+}
+
+const womanCat = {
+    type : "human-cat",
+    name : "woman Cat",
+    gender : "female",
+    legs : 2,
+    hasTail : true,
+    saying : "Meow,Kottans are the best of the best!"
+}
+
+function objectsToString (obj){
+    let objectAsString = "";
+    for (prop in obj){
+        if (prop === "saying"){
+            objectAsString +=prop + " : \"" + obj[prop]+ "\"";
+            break;
+        } else if (prop === "hasTail" || prop === "legs"){
+        objectAsString += prop + " : " + obj[prop] + ", ";
+        continue;
+        }
+        objectAsString += obj[prop] + ", ";
+    }
+    return objectAsString + ";";
+}
+
+print(objectsToString(dog));
+print(objectsToString(cat));
+print(objectsToString(man));
+print(objectsToString(woman));
+print(objectsToString(womanCat));
+
+// ======== OUTPUT ========
+/* Use print(message) for output.
```

Comment:
Please remove all comments and unrelated text.

Reply:
Done.

Reply:
resolve it